web.tf

```diff
@@ -28,6 +28,9 @@ resource "aws_security_group" "acme_web" {
   }
 
  /* ingress {
+   
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0